{{cookiecutter.project_slug}}/.circleci/config.yml

```diff
@@ -134,6 +133,16 @@ jobs:
       - checkout
       - restore_cache:
           key: v1-{{cookiecutter.project_slug}}-py-{% raw %}{{ checksum "requirements/base.txt" }}-{{ checksum "requirements/test.txt" }}-{{ checksum "requirements/local.txt" }}{% endraw %}
+      - run:
+          name: Wait for postgres server
+          command: |
+            for i in `seq 1 10`;
+            do
+              nc -z localhost 5432 && echo Success && exit 0
+              echo -n .
+              sleep 1
+            done
+            echo Failed waiting for Postgres && exit 1
```

Comment:
This is from the CircleCI docs, isn't it? I don't love it, but if it's what they suggest, I can tolerate it.

Reply:
Yeah, that's from: https://circleci.com/docs/2.0/postgres-config/#example-go-app-with-postgresql

But I actually see what I think is a better solution, so I'll try that: https://circleci.com/docs/2.0/databases/#using-dockerize-to-wait-for-dependencies